src/components/company/CompanyConstants.ts

```diff
@@ -0,0 +1,39 @@
+import { CHECK_INTEREST_PROVIDER } from 'Enum';
+import type {
+    CompanySettingsProps,
+    LeadSettingsProps,
+    LmsSettingsProps
```

Comment:
pls rename the file as Constants

Reply:
Moved the file from `src/components/company/CompanyConstants.ts` to `src/components/Constants.ts`